### PR TITLE
Feature#19 styles register page

### LINKED ISSUE #19: Create styles for register page.
# REQUIREMENTS
Create the styles for the page that allows you to select the role with which you want to register in the application.

# Acceptance Criteria
- [ ] Styles for register page.

### PR DESCRIPTION
**Pull request by RobertoCarMed**
# Description
All the styles corresponding to the static content of regsiter page have been added.
Styles have only been added for screen size larger than 1440px.
  
  ## Screenshots 
image

image

resolve #19

### CODE REVIEW

**Review by xFerchoVJ — APPROVED** [review]
LGTM

File: app/assets/stylesheets/pages/register.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+}
+@media(min-width:1440px){
+  @use "./components/register_desktop";
+
+}
+
+@media(min-width:720px){
+  @use "./components/register_movil";
+
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/pages/register.scss:12]: Add a blank line.

File: app/assets/stylesheets/pages/components/_register_desktop.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      img{
+        width: 100%;
+        height: 100%;
+        margin-bottom: 40px;
+      }
+
+    }
+
+  }
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/pages/components/_register_desktop.scss:48]: Add a blank line.

**Review by YilKanda — COMMENTED** [review]
Did you consider mobile views first in your design?

**Review by brayandiazc — APPROVED** [review]

File: app/assets/stylesheets/pages/register.scss (hunk 1 above)
Comment (RobertoCarMed) [reply app/assets/stylesheets/pages/register.scss]: resolved

File: app/assets/stylesheets/pages/components/_register_mobile.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
+.register_page{
+  background-color: #caf0f8;
```
Comment (jclopezdev) [inline app/assets/stylesheets/pages/components/_register_mobile.scss:2]: What do you think of ordering all your style properties alphabetically?

**Review by xFerchoVJ — APPROVED** [review]
LGTM, I recommend you order your style properties alphabetically, it is a good practice!

File: app/assets/stylesheets/pages/components/_register_mobile.scss (hunk 3 above)
Comment (RobertoCarMed) [reply app/assets/stylesheets/pages/components/_register_mobile.scss:2]: In next issue we are resolving

**Review by SaraNavarro — APPROVED** [review]

**Review by SampabloC — APPROVED** [review]

**Pull request merged**